commands/info.js

```diff
@@ -0,0 +1,41 @@
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const embed = require('../utils/embed.js');
+const ms = require('ms');
+const wait = require('util').promisify(setTimeout);
+
+const gitSha =
+	process.env.GIT_SHA || '0ed46333853d1aaab845c769a88845b4fd7b9dde';
+const commitMessage =
+	process.env.COMMIT_MESSAGE ||
+	`Remove Arm images, we no longer need these (#40)
+
+* Remove Arm images, we no longer need these
+* Remove strings that aren't needed`;
+
+const uptime = async () => {
+	if (!process.uptime) await wait(1000);
+	return ms(Math.floor(process.uptime() * 1000), { long: true });
+};
+
+module.exports = {
+	data: new SlashCommandBuilder()
+		.setName('info')
+		.setDescription('Prints metadata.'),
```

Comment:
should we change metadata to info for consistency

Reply:
Changed in [cb69533](https://github.com/Cutie-Club/cutiebot/pull/41/commits/cb69533ba9fee63c62f8560c18779f4cd591488e)